### ISSUE TITLE
argv in calc script

### ISSUE THREAD

**Issue by planet36** [lcn2/calc#10]
```
print version();
argc = argv();
for (i = 0; i < argc; i++) print "argv[": i : '] = "': argv(i) : '"';
```

When the above calc script is invoked like this:
`calc -f test-argv.cal a bb ccc`

This is the output:

> 2.12.7.2
> "a" is undefined
> 
> Error in commands
> 

When the "-s" option is given after the -f arg, it's correct.
```calc -f test-argv.cal -s a bb ccc```

> 2.12.7.2
> argv[0] = "a"
> argv[1] = "bb"
> argv[2] = "ccc"
> 

According to the man page:

>        -f
>               Use of -f implies -s.
> 
>        -s     By default, all calc_cmd args are evaluated and executed.   This
>               flag  will disable their evaluation and instead make them avail‐
>               able as strings for the argv() builtin function.
> 

I thought that by giving "-f" that "-s" was unnecessary.

**Comment by lcn2** [comment]
This is a bug that needs to be addressed.

**Comment by lcn2** [comment]
This bug a a priority for our next stable release.

**Comment by lcn2** [comment]
Hello @planet36,

This has been fixed in release 2.13.0.0.  While this is a pre-release (not a stable release), we have had some good experience in using scripts using this latest release.

Thank you for reporting this and thank you for your patience while we fixed this bug.

chongo (Landon Curt Noll) ^oo^
